sp20/syllabus.yml

```diff
@@ -79,7 +83,7 @@ meeting03:
       datasets: []
       competitions: []
```

Comment:
add comp following the naming scheme: "ucfai-core-sp20-cnns"

Reply:
don't add the competition. `autobot` does that already.